flight/drone/drone.py

```diff
@@ -133,8 +133,7 @@ def arm(self, mode=c.Modes.GUIDED.value):
 
         self._logger.info('Arming...')
         while not self.armed:
-            if self.is_armable:
-                self.armed = True
+            self.armed = True
```

Comment:
Is there a particular reason why this is preferred over checking if the drone is armable?

Reply:
Yeah, so it turns out that drone.vehicle.is_armable is just a bugged value (see https://github.com/dronekit/dronekit-python/issues/695 and https://discuss.ardupilot.org/t/is-armable-parameter-doesnt-change-dronekit-python/21574), or its just not being used right. It always returns False, and so it prevents takeoff.

I had John submit a pull request to add checking for is_armable, but we never actually tested it (not good).